### PR TITLE
Add section on constructing identity input in README

### PR DESCRIPTION
**Pull request by shjortConcordium**
## Purpose

Add section to README to help constructing identityInputs for creating credentials/accounts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -304,6 +305,65 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
 const transactionStatus = await client.getTransactionStatus(transactionHash);
 ```
 
+## Construct identityInput for creating credentials
+
+When creating a new identity, the user will choose an identity provider, create an id-use-data object, which contains the private data to use the identity, and obtain an identity object from the identity provider.
+
+To create accounts/credentials on that identity, this SDK expects an "IdentityInput" object, which contains the identity object, the id-use-data, and the identity provider's information.
+
+Below is an example of how to construct the identityInput, with a plantext id-use-data.json from the [user-cli guide](https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#generate-a-request-for-the-identity-object), and an id-object file.
```
Comment (abizjak) [inline README.md:314]: Perhaps add a subsection heading here to delineate this example from the one below.

File: README.md [hunk 2]
```diff
@@ -304,6 +305,65 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
...
+        ipInfo: idUseData.ipInfo,
+        arsInfos: idUseData.ars,
+    },
+    idCredSecret: idUseData.idUseData.aci.credentialHolderInformation.idCredSecret,
+    prfKey: idUseData.idUseData.aci.prfKey,
+    randomness: idUseData.idUseData.randomness,
+};
+```
+
+Next is an example of how to construct the identityInput for the  _i_-th identity from a mobile wallet export:
```
Comment (abizjak) [inline README.md:344]: And a subsection heading here as well.

**Review by abizjak — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -304,6 +305,70 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
 const transactionStatus = await client.getTransactionStatus(transactionHash);
 ```
 
+## Construct identityInput for creating credentials
```
Comment (orhoj) [inline README.md:308]:
```suggestion
## Construct IdentityInput for creating credentials
```

File: README.md [hunk 4]
```diff
@@ -304,6 +305,70 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
...
+
+When creating a new identity, the user will choose an identity provider, create an id-use-data object, which contains the private data to use the identity, and obtain an identity object from the identity provider.
```
Comment (orhoj) [inline README.md:310]:
```suggestion
When creating a new identity the user will choose an identity provider, create an id-use-data object, which contains the private data to use for the identity, and obtain an identity object from the identity provider.
```

File: README.md [hunk 5]
```diff
@@ -304,6 +305,70 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
...
+To create accounts/credentials on that identity, this SDK expects an "IdentityInput" object, which contains the identity object, the id-use-data, and the identity provider's information.
+
+### Construct from user-cli output:
+
+Below is an example of how to construct the identityInput, with a plantext id-use-data.json from the [user-cli guide](https://github.com/Concordium/concordium-base/blob/main/rust-bins/docs/user-cli.md#generate-a-request-for-the-identity-object), and an id-object file.
+
+```js
+// First we load the files. We assume here that they are available as local files.
+const rawIdUseData = fs.readFileSync(
+    'path/to/id-use-data',
```
Comment (orhoj) [inline README.md:321]:
```suggestion
    'path/to/id-use-data.json',
```

File: README.md [hunk 6]
```diff
@@ -304,6 +305,70 @@ const transactionHash = getCredentialDeploymentTransactionHash(credentialDeploym
...
+    'utf8'
+);
+const rawIdObject = fs.readFileSync(
+    'path/to/id-object',
```
Comment (orhoj) [inline README.md:325]:
```suggestion
    'path/to/id-object.json',
```

**Review by orhoj — CHANGES_REQUESTED** [review]
The mobile wallet export default file extension could also be added.

**Review by orhoj — APPROVED** [review]

**Pull request merged**